test/jdk/java/awt/print/Dialog/PrintDlgPageable.java

```diff
@@ -25,105 +25,118 @@
-                        return Printable.NO_SUCH_PAGE;
-                }*/
-                // Simply draw two rectangles
-                Graphics2D g2 = (Graphics2D)g;
-                g2.setColor(Color.black);
-                g2.translate(pf.getImageableX(), pf.getImageableY());
-                g2.drawRect(1,1,200,300);
-                g2.drawRect(1,1,25,25);
-                System.out.println("print method called "+pi + " Orientation "+pf.getOrientation());
-                return Printable.PAGE_EXISTS;
+        if (pj.printDialog()) {
+            try {
+                PassFailJFrame.log("About to print the data ...");
+                pj.print();
+                PassFailJFrame.log("Printed");
+            }
+            catch (PrinterException pe) {
+                pe.printStackTrace();
+            }
         }
+        passFailJFrame.awaitAndCheck();
+    }
+
+    //printable interface
+    public int print(Graphics g, PageFormat pf, int pi) throws PrinterException {
+
+        // Simply draw two rectangles
+        Graphics2D g2 = (Graphics2D) g;
+        g2.setColor(Color.black);
+        g2.translate(pf.getImageableX(), pf.getImageableY());
+        g2.drawRect(1, 1, 200, 300);
+        g2.drawRect(1, 1, 25, 25);
+        PassFailJFrame.log("print method called " + pi + " Orientation " + pf.getOrientation());
+        return Printable.PAGE_EXISTS;
+    }
 }
 
 class PageableHandler implements Pageable {
 
-        PageFormat pf = new PageFormat();
+    PageFormat pf = new PageFormat();
 
-        public int getNumberOfPages() {
-                return PrintDlgPageable.arg;
-                //return 0;
-        }
+    public int getNumberOfPages() {
+        return PrintDlgPageable.arg;
+    }
 
-        public Printable getPrintable(int pageIndex) {
-                return new PrintDlgPageable();
-        }
+    public Printable getPrintable(int pageIndex) {
+        return new PrintDlgPageable();
+    }
 
-        public PageFormat getPageFormat(int pageIndex) {
-                System.out.println("getPageFormat called "+pageIndex);
-                if (pageIndex == 0) {
-                        pf.setOrientation(PageFormat.PORTRAIT);
-                        System.out.println("Orientation returned from Pageable "+findOrientation(pf.getOrientation()));
-                        return pf;
-                } else {
-                        pf.setOrientation(PageFormat.LANDSCAPE);
-                        System.out.println("Orientation returned from Pageable "+findOrientation(pf.getOrientation()));
-                        return pf;
-                }
+    public PageFormat getPageFormat(int pageIndex) {
+        System.out.println("getPageFormat called " + pageIndex);
+        if (pageIndex == 0) {
+            pf.setOrientation(PageFormat.PORTRAIT);
+            PassFailJFrame.log("Orientation returned from Pageable " + findOrientation(pf.getOrientation()));
+            return pf;
```

Comment:
duplicate code...can be moved outside of `if..else` block.

Reply:
Removed the whole method as logging for printing is not needed for this PageRange check test